app/assets/javascripts/Components/Result/image_viewer.jsx

```diff
@@ -6,15 +6,30 @@ export class ImageViewer extends React.Component {
   constructor() {
     super();
     this.state = {
-      rotation: 0,
-      zoom: 1,
+      rotation: window.start_image_rotation || 0,
+      zoom: window.start_image_zoom || 1,
       heightChange: 0,
       widthChange: 0
     };
   }
 
   componentDidUpdate(prevProps, prevState) {
     this.display_annotations();
+    this.adjustPictureSize();
+    this.rotateImage();
+  }
+
+  componentDidMount() {
+    document.getElementById('image_preview').onload = () => {
+      this.display_annotations();
+      this.adjustPictureSize();
+      this.rotateImage();
+    };
+  }
+
+  componentWillUnmount() {
```

Comment:
Please add a TODO to change this code to store the data within React rather than on `window`.

Reply:
made an issue instead